cpp/test/sparse/dist_coo_spmv.cu

```diff
@@ -64,29 +64,51 @@ class SparseDistanceCOOSPMVTest
   : public ::testing::TestWithParam<
       SparseDistanceCOOSPMVInputs<value_idx, value_t>> {
  public:
+  SparseDistanceCOOSPMVTest() : dist_config(handle) {}
```

Comment:
Are we running each test on a couple different chunk sizes?

Reply:
Hmmm yeah, we should do that. I'll add a test for chunk size of 3